src/views/Auth/Auth.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { useAuth } from '../../hooks/useAuth';
+import { signInUser, signUpUser } from '../../services/user';
+
+import './Auth.css';
+
```

Comment:
I believe we have to import style from './Auth.css' in order for our css to load.